README.md

```diff
@@ -19,7 +19,7 @@ This module is designed to setup a full katello server, including candlepin, pul
 
 ## Limitations
 
-* EL7 (RHEL7 / CentOS 7)
+* Supports EL7 (RHEL7 / CentOS 7) and EL8 (RHEL8 / CentOS 8)
```

Comment:
How about we refer users to `operatingsystem_support` in `metadata.json` for the exact OS support?

Reply:
> Pro: one less thing to update whenever the supported OSes change (but how often does that _really_ happen)

This is what I'm mostly after. As you can see, it doesn't happen enough that we tend to mess it up. Especially since not all modules have it in their README. However, our CI relies on metadata.json so that's guaranteed to be correct. So it would be my preference to refer users there.

Reply:
I've just pushed another commit with an updated text.